scripts/tool_shed/build_ts_whoosh_index.py

```diff
@@ -191,6 +183,23 @@ def get_repos( sa_session, path_to_repositories ):
                              tools_list=tools_list ) )
     return results
 
+def load_one_dir( path ):
+    tools_in_dir = []
+    tool_elems = load_tool_elements_from_path( path )
+    if tool_elems:
+        for elem in tool_elems:
+            root = elem[1].getroot()
+            if root.tag == 'tool':
+                tool = {}
+                if root.find( 'help' ) is not None:
+                    tool.update( dict( help=root.find( 'help' ).text ) )
+                if root.find( 'description' ) is not None:
+                    tool.update( dict( description=root.find( 'description' ).text ) )
+                tool.update( dict( id=root.attrib.get( 'id' ),
+                                   name=root.attrib.get( 'name' ),
+                                   version=root.attrib.get( 'version' ) ) )
+                tools_in_dir.append( tool )
+    return tools_in_dir
 
```

Comment:
Missing the second blank line for flake8 (this directory is in the blacklist but this file is otherwise clean).